pages/Configuring/Window-Rules.md

```diff
@@ -27,65 +28,77 @@ windowrule=move 0 0,title:^(Firefox)(.*)$
+| rounding \[x\]                | forces the application to have X pixels of rounding, ignoring the set default (in `decoration:rounding`). Has to be an int.                                                                                                                                                                                            |
+| noblur                        | disables blur for the window                                                                                                                                                                                                                                                                                           |
+| nofocus                       | disables focus to the window                                                                                                                                                                                                                                                                                           |
+| forceinput                    | forces an XWayland window to receive input, even if it requests not to do so. (Might fix issues like e.g. Game Launchers not receiving focus for some reason)                                                                                                                                                          |
+| pin                           | pins the window _note: floating only_                                                                                                                                                                                                                                                                                  |
+| noanim                        | disables the animations for the window                                                                                                                                                                                                                                                                                 |
+
+### Example Rules
+
 ```ini
 windowrule = move 100 100,^(kitty)$
 windowrule = animation popin,^(kitty)$
 windowrule = noblur,^(firefox)$
 ```
 
+## Notes
+
 {{< hint type=tip >}}
 
-Opacity is always a PRODUCT of all opacities. E.g. active_opacity to
+Opacity is _always_ a PRODUCT of all opacities. E.g. active*opacity to
 0.5 and windowrule opacity to 0.5 will result in a total opacity 0.25. You are
 allowed to set opacities over 1, but any opacity product over 1 will cause
-graphical glitches. E.g. 0.5 * 2 = 1, and it will be fine, 0.5 * 4 will cause
+graphical glitches. E.g. 0.5 \_2 = 1, and it will be fine, 0.5* 4 will cause
```

Comment:
This was intended to be *.

Reply:
Probably auto-changed by neovim. I usually know how math works.